README.md

```diff
@@ -174,3 +175,108 @@ Explanation of the switches:
 
 + `-e` <ELEMENT> : MD element type for search, e.g. _doi_ or _creator_
 + `-q` <QUERY> : MD content to start the search
+
+
+# Adding new parsers
+
+## Introduction
+
+The extractor tool uses python classes as parser modules that can process specific formats as described above.
+
+In order to add a new custom parser, you have to write a parser class and add it to the directory in the following location: `o2r-meta\parsers`.
+
+Note that future versions of o2r-meta might be able to register all parsers based on the files in the `parsers` directory. Until then it is necessary to register a new parser in the function `register_parsers` of `o2r-meta\extract\metaextract.py`.
+
+The use of classes as parsers is currently still implemented in a rather naïve way. However, in order to be able to take advantage of non static methods, all parsers are designed as classes already. Also a governing class that provides inheritance features should be the goal of a better extractor.
+
+
+## Structure of your parser Python class
+
+_In this example we will call your new parser file `parse_abc.py` and the class in that file `ParseAbc`._
+
+Here is a extensively commented template for your parser file:
```

Comment:
Strike ~~extensively~~, it makes me think as a reader what I did wrong that I need _extensive_ comments.

Reply:
Okay! (Though I do not share your feelings about that word)